go.mod

```diff
@@ -35,9 +35,9 @@ require (
 	github.com/imdario/mergo v0.3.12
 	github.com/mattn/go-isatty v0.0.12 // indirect
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v0.0.0-20210921141042-49b82c6d1ca9
+	github.com/openshift/api v0.0.0-20210924154557-a4f696157341
```

Comment:
Also need to pin this version by updating openshift/api version in replace section https://github.com/openshift/machine-config-operator/blob/master/go.mod#L38

Reply:
Done